app/controllers/schools/cohort/setup_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Schools
+  module Cohort
```

Comment:
Hmm not sure this is a great module name. I'm guessing that this caused issues in the code (hence the addition of `::` to `Cohort` when referring to the model?